### ISSUE TITLE
Book of Daring is sticky

### ISSUE THREAD

**Issue by JKingweb** [larsiusprime/tdrpg-bugs#1093]
Operating System: Windows 10
Browser/Download: Download
Demo/Full?: Full Version (GOG)
Version #: 2.0.1 b

Steps to reproduce:
1. Begin Challenge 5 with Book of Daring equipped
2. Drop Ketta somewhere
3. Recall Ketta
4. Quit challenge
5. Start any battle

What I expected to happen:

I should be able to summon Ketta again after starting a subsequent battle

What actually happens:

I'm unable to summon Ketta again, ever, unless I return to the title screen.  Unequipping the Book of Daring makes no difference. Restarting the challenge (rather than quitting) yields the same outcome.

**Comment by ManDude** [comment]
What resolution are you playing at?

**Comment by JKingweb** [comment]
1920x1080 full-screen

**Comment by codingthat** [comment]
@JKingweb Still seeing this?  I'm playing at the same resolution, Steam DQ1 2.2.2_1/30/2018, Linux Mint 18 64-bit, but I couldn't reproduce it with these steps.  (Any battle = I tried Challenge 5 again, and also one from the map; in both I could place Ketta still.  In the initial battle it functioned correctly in that once I recalled Ketta I couldn't place her again, but only for that battle.)  This was on NG+ for me.
